xarray/backends/api.py

```diff
@@ -295,6 +295,9 @@ def maybe_decode_store(store, lock=False):
         elif engine == 'pynio':
             store = backends.NioDataStore(filename_or_obj,
                                           autoclose=autoclose)
+        elif engine == 'pnc':
```

Comment:
I would slightly rather use the full name "pseudonetcdf" than an abbreviation like "pnc". We don't use abbreviations for any of the other backend names (to be fair, yes, they are also shorter).

Reply:
Done.